### ISSUE TITLE
No hotspot in wallet

### ISSUE THREAD

**Issue by Tomval1** [helium/wallet-app#261]
No hotspot in the Helium wallet at Collectebles. What can be wrong? Miner works properly.

**Comment by Tomval1** [comment]
Lucky you. I did the same but this doesn't work for me 😐. Thanks anyway

**Comment by Perronef5** [comment]
This should be fixed on version 2.0.7. If the app still shows no hotspots, please try going into settings and click `migrate wallet`

**Comment by Tomval1** [comment]
Yes, it's Merry Black Ant

Dne st 24. 5. 2023 22:15 uživatel Bryan ***@***.***> napsal:

**Comment by ChewingGlass** [comment]
@Tomval1 I'm showing that the hotspot belongs to wallet `EGkMPV5Q3gQqjQwvpS6dzgmiGZCEJVT9vhBHLw7ZewzD`

It was transferred on April 26 in this tx https://explorer.solana.com/tx/3Q1298kGBKfBaWaZuMj1whz832VJxM3aig9iVUtwhNk6eJywQS6WZMjxNpaBXUdJdMxmZs5RB7CPz3geqL85iXmD from wallet `GtMgq8ZxvbHujZPc5odRkKDVWGb8SsfkNmrD51V57bgJ`

**Comment by Tomval1** [comment]
This wallet is not mine: EGkMPV5Q3gQqjQwvpS6dzgmiGZCEJVT9vhBHLw7ZewzD

Is there any chance to fix it? Hard miner reset or Upload new firmware into
the miner or something like this?

Thanks

Dne út 20. 6. 2023 17:06 uživatel Noah Prince ***@***.***>
napsal:

**Comment by ChewingGlass** [comment]
There is not, unfortunately. Because it's crypto, transfers are permanent. This could be related to https://github.com/helium/wallet-app/issues/360

Where did you get that address? Can you ask them to transfer it back to you?

**Comment by Tomval1** [comment]
This happened when HTN switched to SOL network. In my wallet appeared NFT
which later Helium development team announced as scam NFT.

I didn't know that I clicked on it and since that time my miner
disappeared. I think my miner was stolen to that address you mentioned.

There was a bug in the Hellium wallet, but who cares.

So I turned my miner off and apparently no chance to fix it

Dne st 21. 6. 2023 19:34 uživatel Noah Prince ***@***.***>
napsal:
